### DISCUSSION TITLE
Tweak Scale fata error

### DISCUSSION THREAD

**Discussion by miqt** [TweakScale/TweakScale#338] [Support] [answered]
Hello @Lisias, so today i am playing KSP with some mods on when suddenly when i opened my KSP it send a TweakScale fatal error.
Here is my KSP.log
[Uploading KSP.log…]()

**Comment by Lisias** [comment]
Hi, @miqt !

Unfortunately, you hit "Comment" before finishing the upload. Please try again but, this time, wait for the `uploading KSP.log` text to turn into `KSP.log` only, the hint that the uploading was completed!

In the mean time, and trying to guess based on the most common cases, did you installed the TweakScale Companion ÜberPacket? Some 3rd parties have some pretty outdated patches, and I fixed them on the Companion. Git it a try, just in case: https://github.com/TweakScale/Companion/releases

**Reply by miqt** [reply]
Ok I will try it

**Reply by miqt** [reply]
and here is my KSP.log
[KSP.log](https://github.com/user-attachments/files/17005000/KSP.log)

**Comment by Lisias** [comment]
Hummm... Interesting...

```
[LOG 14:55:40.337] [TweakScale] ERROR: **FATAL** Part KIS.bomb1 (K4 Standard Explosives) has a fatal problem due having duplicated properties - s
ee issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
[LOG 14:55:40.337] [TweakScale] ERROR: **FATAL** Part kis.concreteBase1 (CB1 Ground Base) has a fatal problem due having duplicated properties -see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
[LOG 14:55:40.337] [TweakScale] ERROR: **FATAL** Part KIS.Container1 (SC-62 Portable Container) has a fatal problem due having duplicated properties - see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
[LOG 14:55:40.338] [TweakScale] ERROR: **FATAL** Part KIS.Container2 (ILC-18k Container) has a fatal problem due having duplicated properties - see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
[LOG 14:55:40.338] [TweakScale] ERROR: **FATAL** Part KIS.Container3 (ISC-6K Container) has a fatal problem due having duplicated properties - see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ). at error:0
```

It's some time since the last time KIS was involved on something.

**Comment by Lisias** [comment, marked as answer]
Hi, @miqt !!!

I found the problem! You installed the Companion twice! See:

```
[LOG 14:51:25.559] Applying update TweakScaleCompanion/KIAS/patches/KIS/KIS-Utility_TweakScale/@PART[KIS_bomb1]:NEEDS[KIS,TweakScale]:FOR[TweakScaleCompanion_KIAS] to KIS/Parts/bomb1/part.cfg/PART[KIS_bomb1]
[LOG 14:51:25.575] Applying update TweakScaleCompanion-2024.07.13.0/GameData/TweakScaleCompanion/KIAS/patches/KIS/KIS-Utility_TweakScale/@PART[KIS_bomb1]:NEEDS[KIS,TweakScale]:FOR[TweakScaleCompanion_KIAS] to KIS/Parts/bomb1/part.cfg/PART[KIS_bomb1]
```
There's a copy on `<KSP>/GameData/Companion`, that it's the right place, but also on `<KSP>/GameData/TweakScaleCompanion-2024.07.13.0/GameData/TweakScaleCompanion` what's pretty wrong!

Delete the directory `TweakScaleCompanion-2024.07.13.0` that it's now on your `GameData` and at least this specific problem will be fixed!

Cheers!

**Reply by miqt** [reply]
OK Thankyou

**Reply by Lisias** [reply]
Let me know in way or another!

Cheers!

**Reply by miqt** [reply]
Sorry to disturb again. But the error still appear.

**Reply by Lisias** [reply]
Send me another `KSP.log`.
